api/database/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.3'
+
+services:
+  db:
+    build: ./db
+    container_name: postgres
+    ports:
+      - "5431:6666"
+    environment:
+      - POSTGRES_USER=SADashboard
```

Comment:
This user and database need to be created in an entrypoint script if they're to be used by the Docker Compose setup.

Reply:
Just realized my comment was incorrect. The Postgres image is configured to watch for these environment variables when it configures itself.